bigtable-client-core-parent/bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/TestBigtableTableAdminClientWrapper.java

```diff
@@ -106,7 +106,7 @@ public void testCreateTableAsync() throws Exception{
 
     when(mockAdminClient.createTableAsync(request.toProto(PROJECT_ID, INSTANCE_ID)))
         .thenReturn(immediateFuture(createTableData()));
-    ListenableFuture<Table> response = clientWrapper.createTableAsync(request);
+    ApiFuture<Table> response = clientWrapper.createTableAsync(request);
```

Comment:
Please use `Future` instead for all of the tests.  The tests shouldn't care about `ListenableFuture` vs. `ApiFuture`

Reply:
Updated Test class to use `Future`.